components/RecipeForm.js

```diff
@@ -149,9 +177,14 @@ export default function RecipeForm({ onSubmit, onChange, imageUrl }) {
             name="instructions"
             required
             aria-label="add instructions for creating the recipe"
+            defaultValue={data?.instructions}
           />
           <StyledH2>Video</StyledH2>
-          <StyledInput type="link" name="youtubeLink"></StyledInput>
+          <StyledInput
+            type="link"
+            name="youtubeLink"
+            defaultValue={data?.youtubeLink}
+          ></StyledInput>
```

Comment:
this can be self-closing:

```suggestion
          <StyledInput
            type="link"
            name="youtubeLink"
            defaultValue={data?.youtubeLink}
          />
```